common/src/db/mod.rs

```diff
@@ -167,6 +167,15 @@ impl Database {
         log::debug!("bootstrap to {}", url);
         let db = sea_orm::Database::connect(url).await?;
 
+        let install_extension_db_result = db
+            .execute(Statement::from_string(
+                db.get_database_backend(),
+                format!(
+                    "CREATE EXTENSION IF NOT EXISTS \"{}\";",
+                    "pg_stat_statements"
+                ),
+            ))
+            .await;
```

Comment:
I think this misses a `?` at the end. Which expands into something like:

```rust
let result = match result {
  Ok(result) => result,
  Err(err) => return Err(err),
}
```

Which is a little bit like "ok or throw". But only works in function which return a `Result`. Also the error of the expression must implement `Into` to the error type of the function's result.

When not using the result (not assigning it), I would expect `cargo check` to warn about this. In this case it looks to me as if it gets assigned (which eliminates the warning) but then doesn't evaluate the result. If you don't need to result value, you can just use `?` and don't assign it. That also counts as "used".

Reply:
ah I probably missed off, thx!